Sources/VLCHTTPConnection.m

```diff
@@ -142,31 +142,20 @@ - (BOOL)fileIsInDocumentFolder:(NSString*)filepath
 
 - (NSObject<HTTPResponse> *)_httpGETThumbnailForPath:(NSString *)path
 {
-    NSString *filePath = [[path stringByReplacingOccurrencesOfString:@"/thumbnail/" withString:@""] stringByAddingPercentEncodingWithAllowedCharacters:NSCharacterSet.URLFragmentAllowedCharacterSet];
-    filePath = [filePath stringByReplacingOccurrencesOfString:@".png" withString:@""];
-
-    NSManagedObjectContext *moc = [[MLMediaLibrary sharedMediaLibrary] managedObjectContext];
-    if (moc) {
-        NSPersistentStoreCoordinator *psc = [moc persistentStoreCoordinator];
-        if (psc) {
-            NSManagedObject *mo = nil;
-            @try {
-                mo = [moc existingObjectWithID:[psc managedObjectIDForURIRepresentation:[NSURL URLWithString:filePath]] error:nil];
-            }@catch (NSException *exeption) {
-                return [[HTTPErrorResponse alloc] initWithErrorCode:404];
-            }
+    NSString *filePath = [[path stringByReplacingOccurrencesOfString:@"Thumbnail/" withString:@""] stringByAddingPercentEncodingWithAllowedCharacters:NSCharacterSet.URLFragmentAllowedCharacterSet];
```

Comment:
Nit-picking: From a small test locally, I believe it should be `/Thumbnail/`.

It is not a big deal since the extra `/` at the beginning will be ignored.

Reply:
I'll try again but I remember faintly that it didn't work when I removed the /